assignments/B-02-81/app/controllers/admin/admin_user.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * This will contain operations related to user model
```

Comment:
Use proper comments like . This will contain admin operation related to user. Remember it for future reference